bevy_asset_loader/examples/progress_tracking.rs

```diff
@@ -49,12 +54,14 @@ struct TextureAssets {
     female_adventurer: Handle<TextureAtlas>,
 }
 
-fn track_fake_long_task(time: Res<Time>, progress: Res<ProgressCounter>) {
-    if time.seconds_since_startup() > 1. {
-        info!("done");
-        progress.manually_track(true.into());
-    } else {
-        progress.manually_track(false.into());
+fn track_fake_long_task(time: Res<Time>, progress: Res<ProgressCounter>, mut printed: Local<bool>) {
```

Comment:
I actually find the old version of this method nicer to read/understand. The `printed` local should not be needed as long as the assets all load before the task (which really should be the case for a 3s task :crossed_fingers:), because the state will be changed directly after the task completes.

Reply:
Most likely, you are right)